src/olympia/reviewers/tests/test_views.py

```diff
@@ -1491,19 +1521,17 @@ def test_results_two_versions(self):
         version2 = self.addons['Nominated Two'].versions.all()[0]
         file_ = version2.files.get()
 
-        # Versions are ordered by creation date, so make sure they're set.
-        past = self.days_ago(1)
-        version2.update(created=past, nomination=past)
-
-        # Create another version, v0.2, by "cloning" v0.1.
+        # Create another version for Nominated Two, v0.2, by "cloning" v0.1.
```

Comment:
Nit: I find this comment a bit confusing because it says "Create another version", but we already have `version2`, which we've created above, and all we're doing is updating some of its properties. Would it be more accurate to say "Update version 2..."?

Reply:
by setting the `pk` to `None` and saving it again, we create a different version (cloning properties of the instance we started from)